src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,213 @@
+# Goals and scope
+
+Derivatives are outputs of common processing pipelines, capturing data and
+meta-data sufficient for a researcher to understand and (critically) reuse those
+outputs in subsequent processing. Standardizing derivatives is motivated by use
+cases where formalized machine-readable access to processed data enables higher
+level processing.
+
+## Storage of derived datasets
+
+Derivatives can be stored/distributed in two ways:
+
+1.  Under a `derivatives/` subfolder in the root of the source BIDS dataset
+    folder to make a clear distinction between raw data and results of data
+    processing. Each pipeline has a dedicated directory under which it stores
+    all of its outputs. There are few restriction on the directory name; it is
+    RECOMMENDED to use the format `<pipeline>-<variant>` in cases where it is
+    anticipated that the same pipeline will output more than one variant (e.g.,
+    `AFNI-blurring`, `AFNI-noblurring`, etc.). For the sake of consistency, the
+    subfolder name needs to be a substring of `PipelineDescription.Name` field
+    in the `dataset_description.json` (see below).
+
+    For example:
+
+    ```Text
+    <dataset>/derivatives/fmripreprocess-v1/sub-0001
+    <dataset>/derivatives/spm/sub-0001
+    <dataset>/derivatives/vbm/sub-0001
+    ```
+
+1.  As a standalone dataset independent of the source (raw or derived) BIDS
```

Comment:
Once #178 is merged this passage and/or `/src/01-common-principles.md` should comment on the status of a folder `derivatives` that sits at the same level as a root BIDS dataset.  We don't _have_ to do anything, as this `01-introduction.md` passage makes clear that derivatives can be a stand-alone datatset, but #178 presently offers this as a structure:

```Text
my_dataset/
  sourcedata/
    ...
  rawdata/
    dataset_description.json
    participants.tsv
    sub-01/
    sub-02/
    ...
  derivatives/
    ...
```

which then begs the question: Is there any special relationship between `my_dataset/rawdata` and `my_dataset/derivatives`?  Are there implied links between the latter and the former? Or should they be explicit?

Voluminous text is not needed, but an acknowledgment of the setting in`/src/01-common-principles.md` (if merged before this PR) would improve clarity.

Reply:
This link could be provided by the means of `SourceDatasets`: https://github.com/bids-standard/bids-specification/pull/109/files#diff-0361044f2e50dc58ce5ac1949c9cf93eR68

Reply:
Resolved in https://github.com/bids-standard/bids-specification/pull/109/commits/3878971f18f4f6871c5e1e33464364903b4bbbcc